riscv/plic.cc

```diff
@@ -340,17 +340,15 @@ bool plic_t::load(reg_t addr, size_t len, uint8_t* bytes)
     }
   }
 
-  if (ret) {
-    memcpy(bytes, (uint8_t *)&val, len);
-  }
+  read_little_endian_reg(val, addr, len, bytes);
```

Comment:
09db15baf422b0af769effe3ad5f09b722f619cf contains only dead code, which impairs `git bisect`; I'd squash it with this commit.

Reply:
Ack.

Reply:
For my edification, why do commits like this impair bisect?  Is it merely that a longer history makes it take longer to bisect, or is there something more subtle that I'm missing?

Reply:
If there was a bug in the dead code, `git bisect` would point to the following commit (where the dead code was first executed) instead of the commit where the bug was actually introduced.

Reply:
Thanks for the explanation.  I think we should run with your preferred approach as a guideline, not a hard-and-fast rule.  Sometimes, the dead code is substantial enough that separating it into its own commit is justifiable on the basis of reducing cognitive load.  (This isn't such a case.)